azure-pipelines.yml

```diff
@@ -4,7 +4,9 @@
 # https://aka.ms/yaml
 
 trigger:
-- main
+  branches:
+    include:
+      - "*"
```

Comment:
not all branches support all builds listed below, and kvmtest.

Reply:
we should consider to do batch as well. https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#batching-ci-runs

Reply:
it is better just enable master and 202012 for now. 201911 will have kvm test failure.